docs/getting-started.rst

```diff
@@ -0,0 +1,74 @@
+===============
+Getting Started
+===============
+
+Installing Prerequisite Software
+--------------------------------
+
+SALib requires `NumPy <http://www.numpy.org/>`_, `SciPy <http://www.scipy.org/>`_,
+and `matplotlib <http://matplotlib.org/>`_ installed on your computer.  Using
+`pip <https://pip.pypa.io/en/stable/installing/>`_, these libraries can be
+installed with:
+
+::
+
+    pip install numpy
+    pip install scipy
+    pip install matplotlib
+    
+Installing SALib
+----------------
+
+To install the latest stable version of SALib using pip, run the following
+command:
+
+::
+
+    pip install SALib
+    
+To install the latest development version of SALib, run the following
+commands.  Note that the development version may be unstable and include bugs.
+We encourage users use the latest stable version.
+
+::
+
+    git clone https://github.com/SALib/SALib.git
+    cd SALib
+    python setup.py install
+
+Testing Installation
+--------------------
+
+To test your installation of SALib, start a new interactive Python session
+and copy/paste the code below.
+
```

Comment:
You can also run the SALib tests from the command line:
    python setup.py test